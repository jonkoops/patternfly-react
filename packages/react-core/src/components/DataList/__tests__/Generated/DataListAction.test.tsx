/**
 * @vitest-environment jsdom
 */
import { render } from '@testing-library/react';
import React from 'react';
import { expect, it } from 'vitest';

import { DataListAction } from '../../DataListAction';

it('DataListAction should match snapshot (auto-generated)', () => {
  const { asFragment } = render(
    <DataListAction
      children={<div>ReactNode</div>}
      className={"''"}
      id={'string'}
      aria-labelledby={'string'}
      aria-label={'string'}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
