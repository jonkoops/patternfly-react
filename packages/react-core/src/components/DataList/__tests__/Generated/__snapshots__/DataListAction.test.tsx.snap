// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DataListAction should match snapshot (auto-generated) 1`] = `
<DocumentFragment>
  <div
    class="pf-v5-c-data-list__item-action ''"
  >
    <div>
      ReactNode
    </div>
  </div>
</DocumentFragment>
`;
